hawkular-inventory-bus/src/main/java/org/hawkular/inventory/bus/MessageSender.java

```diff
@@ -65,4 +73,18 @@ public void send(Interest<?, ?> interest, Object inventoryEvent) {
     private String firstLetterLowercased(String source) {
         return Character.toLowerCase(source.charAt(0)) + source.substring(1);
     }
+
+    private ProducerConnectionContext init() throws JMSException, NamingException {
+        InitialContext namingContext = new InitialContext();
```

Comment:
`InitialContext` should be `close()`d afer usage, because it can leak resources.